showcases/vue-showcase/package.json

```
@@ -12,7 +12,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@db-ui/foundations": "*",
+    "@db-ui/foundations": "file:../../packagesi/foundations",
```

@nmerget Is there a typo in the path ("packages**i**")?
```suggestion
    "@db-ui/foundations": "file:../../packages/foundations",
```

---

fixed